src/cryptsetup/cryptsetup-generator.c

```diff
@@ -903,18 +903,16 @@ static int add_proc_cmdline_devices(void) {
                 if (!device)
                         return log_oom();
 
-                r = create_disk(d->name,
+                RET_GATHER(ret, create_disk(d->name,
                                 d->datadev ?: device,
                                 d->keyfile ?: arg_default_keyfile,
                                 d->keydev,
                                 d->headerdev,
                                 d->options ?: arg_default_options,
-                                "/proc/cmdline");
-                if (r < 0)
-                        return r;
+                                "/proc/cmdline"));
```

Comment:
Please re-indent the arguments.

Reply:
I don't think any re-indentation helps to make this more readable. I've moved the `RET_GATHER` to a separate statement instead; that looks better to me at least.